### ISSUE TITLE
Mismatch of source file paths

### ISSUE THREAD

**Issue by joaosantos15** [dmuhs/mythx-cli#133]
* MythX CLI version: latest

### Description

Upon inspecting the JSON submitted to the mythx api, we noticed there was a mismatch of source file paths, which prevented issues location decoding from being performed.

Source List included:
```json
"sourceList" : [ 
            "UniswapV2ERC20.sol", 
             ...
]
```

Matching entry on the sources object:
```json
"sources" : "{"contracts/UniswapV2ERC20.sol":{"ast ... }}
```

Notice how the `UniswapV2ERC20` in `sources` in prefix by `contracts` but not in the `sourceList`.

**Comment by dmuhs** [comment]
This is most likely an issue in the path trimming, which is meant to improve user privacy: https://github.com/dmuhs/mythx-cli/blob/master/mythx_cli/analyze/util.py#L16

Trimming is done by a common prefix - I'll have to dig a bit deeper as the prefix seems to be the same at all points.

**Comment by dmuhs** [comment]
This has been fixed in #134. The issue was a relative path that snuck in and resulted in a prefix replacement not taking effect. I have also released the patch under version 0.6.15.
